### ISSUE TITLE
Hand over project to somebody else

### ISSUE THREAD

**Issue by untitaker** [untitaker/python-webuntis#3]
I'm no longer interested in maintaining this package since I have no real-life use for it. Anybody who thinks they're up for this task, please comment here.

**Comment by maphy-psd** [comment]
Hi! Ich würde gerne die Repository übernehmen.

I would like to take over the repository.

Greets

**Comment by untitaker** [comment]
Cool! Some rules I never wrote down:

* releases should be marked with tags
* `make release` will take care of uploading to PyPI

I also requested a repo transfer. What is your name on PyPI?

**Comment by untitaker** [comment]
>When should I accept the request repo transfer? Do you need access to the repo for some tasks? 

No, I will become a collaborator automatically. You can accept at any moment.

**Comment by untitaker** [comment]
I added you as owner on PyPI

**Comment by untitaker** [comment]
There's still some things left, most notably the "UNMAINTAINED" warnings in the repo description and the README should probably be removed. :)

**Comment by maphy-psd** [comment]
Yah, thanks for the info. I changed it. :)

**Comment by untitaker** [comment]
Wouldn't it make sense to remove the entire alert in the README? (this is your package now btw, feel free to turn everything upside down)

**Comment by Yepoleb** [comment]
Seems like the project is more dead now than it was with the old maintainer.

**Comment by untitaker** [comment]
I would recommend adding @Yepoleb as maintainer then...

**Comment by maphy-psd** [comment]
@Yepoleb Au! I didn't get a notice of your pull request.

At the moment i test some new code on a private repo. During the christmas break i can push some new code.

@untitaker i think it would make sense.

**Comment by Yepoleb** [comment]
I'm not a regular user, so I wouldn't be able to properly maintain the project. Looking at a few issues and PRs from time to time is fine for me though.

**Comment by pintman** [comment]
Is the repo stil vital? Some pull requests are still in the line waiting for action.

**Comment by untitaker** [comment]
Let's make an organization for this project if nobody has the time to be the main maintainer

On 16 June 2017 20:40:43 GMT+02:00, Marco Bakera <notifications@github.com> wrote:
>Is the repo stil vital? Some pull requests are still in the line
>waiting for action.
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/maphy-psd/python-webuntis/issues/3#issuecomment-309103563

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by maphy-psd** [comment]
Sorry for my silence. I was busy the last month. I will check the PR this weekend.

@untitaker I think a organization is useful.

**Comment by untitaker** [comment]
Let's do it then. Since you are the only admin/owner, only you can do that.

**Comment by maphy-psd** [comment]
The question is: Who wants to help in this repo? @pintman @Yepoleb

**Comment by TobiasFranek** [comment]
@maphy-psd I would love to help, but i don't now the repo very well. I also have written a similar package but in php.

**Comment by Yepoleb** [comment]
I'm not using WebUntis anymore, so my interest has dropped to zero. I'll also lose my account at some point.

**Comment by untitaker** [comment]
It also is irrelevant who wants to help. Just add anybody to the org who showed
interest at some point. That's the only way this repo will survive.

**Comment by maphy-psd** [comment]
I have invited @pintman @TobiasFranek @Yepoleb and @AugustH as a Collaborator. So when I have the answers then we can make a plan for the repo.

**Comment by maphy-psd** [comment]
Ok..now we can make a future plan for this repo. Tomorrow I will create a issues with my ideas.

**Comment by untitaker** [comment]
Just create the organization! That's all that can be done...

**Comment by maphy-psd** [comment]
check!

**Comment by TheGreyDiamond** [comment]
So,...... Is this now completely dead? It would be very sad if so

**Comment by AugustH** [comment]
no, i did some work lately. it should be up to date with the official api.

**Comment by pintman** [comment]
I would call it a slow but steady development process. :)

**Comment by cofw2005** [comment]
Hello friends,
after a few years, here might comes a request to maintain this package again. Due the WebUntis changes the server address recently, the new address may contains "-" symbol, this caused error "Not a valid hostname". Someone has investigate and located the root cause was this code in userinput.py.
```
    if not re.match(r'^[a-zA-Z0-9.:-_]+$', urlobj.netloc):
        # That's not even a valid hostname
        raise ValueError('Not a valid hostname')
```
The symbol "-" was not considered as a valid letter. Detailed discussion you can find in this link:
https://github.com/JonasJoKuJonas/homeassistant-WebUntis/issues/231#issuecomment-3561582078

Could anyone of you owners make an update of the python package? Thanks a lot in advance!
@maphy-psd @AugustH

**Comment by IceWizard7** [comment]
Hello everyone,

I’m very interested in this project and would love the opportunity to contribute as a maintainer.
I currently have an open PR (#66) where I’m working on renewing the packaging. In the future, I also plan to upgrade unit tests (#5) to improve stability and maintainability.

I also noticed that many people here speak German — I'm from Austria, so that wouldn’t be a problem for me.

Greetings,
Ice

@maphy-psd I’d be grateful if you could add me.

**Comment by AugustH** [comment]
just seen your PR, but: the project is "finished": the given API is fully implemented and the API is officially deprecated. 
- "maintainability" is not really a problem as there will no changes in the API 
- is there any gain by new build/packaging?
- is there any gain by new unit tests? there is only little logic in the code, it is mostly passing (the right) parameters from and to the API
- what do you mean by "improve stability"?

Greetings

**Comment by untitaker** [comment]
I think you should feel free to open those PRs or file issues containing proposals and they can be discussed independently. I feel that this thread is being revived with various ideas about what the package could be that are not related to staffing. I therefore will close this thread.

If there is a problem such as the API breaking due to changes in hostnames, just like with everything else, file a separate issue.
